prow/cmd/hook/Dockerfile

```diff
@@ -15,7 +15,7 @@
 FROM alpine:3.4
 MAINTAINER spxtr@google.com
 
-RUN apk add --no-cache ca-certificates && update-ca-certificates
+RUN apk add --no-cache git ca-certificates && update-ca-certificates
```

Comment:
Did you think through using a sidecar like https://github.com/kubernetes/git-sync instead?